test/audits/link-with-unclear-purpose.js

```diff
@@ -132,3 +132,13 @@ test('a hidden link should not be run against the audit', function() {
       rule.run({scope: fixture}),
       { result: axs.constants.AuditResult.NA });
 });
+
+test('an anchor tag without href attribute is ignored', function() {
+    var fixture = document.getElementById('qunit-fixture');
+    var a = fixture.appendChild(document.createElement('a'));
+
+    var rule = axs.AuditRules.getRule('linkWithUnclearPurpose');
+    deepEqual(
+      rule.run({scope: fixture}),
```

Comment:
Would fix this but it's consistent with the rest of the file, so I don't think it's worth it.